rosa_getting_started_sts/rosa-sts-deleting-access-cluster.adoc

```diff
@@ -5,8 +5,14 @@ include::modules/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-Delete access to a {product-title} (ROSA) cluster using the `rosa` command-line.
+Access to a {product-title} (ROSA) cluster is set up through the identity provider (IDP) accounts of users.
+
+You can revoke the administrator access of users so that they can access the cluster without administrator privileges. To remove the administrator access for a user, you must revoke the `dedicated-admin` or `cluster-admin` privileges.
+
+You can also remove full access of users so that they cannot access the cluster. To remove full access of a user, you must remove the user through the OpenShift Cluster Manager (OCM) console in addition to revoking the administrator privileges.
```

Comment:
Some comments with the one in modules/rosa-delete-users.adoc

Reply:
Same comment as above regarding removing full access.

Reply:
Incorporated this feedback.